main.js

```diff
@@ -0,0 +1,56 @@
+let bookList;
+
+// remove book from booklist array
+function removeBook(book) {
+  bookList.splice(bookList.indexOf(book), 1);
+}
+
```

Comment:
- [ ] Kindly check the functionality of your localStorage since when I'm adding a book or deleting one, it's not updated until I refresh the page.